examples/csharp/HelloworldXamarin/README.md

```diff
@@ -6,23 +6,26 @@ EXPERIMENTAL ONLY
 Support of the Xamarin platform is currently experimental.
 The example depends on experimental Grpc.Core nuget package that hasn't
 been officially released and is only available via the [daily builds](https://packages.grpc.io/)
-source.
+source. 
+NOTE: To downlaod the package please manually [download](https://packages.grpc.io/archive/2018/07/a3b54ef90841ec45fe5e28f54245b7944d0904f9-d24c85c7-32ed-4924-b9af-80e7a4aeb34d/index.xml) the .nupkg file into a local directory. Then add a nuget source that points to that directory (That can be done in [Visual Studio](https://docs.microsoft.com/en-us/nuget/tools/package-manager-ui#package-sources) or Visual Studio for Mac via "Configure nuget sources"). After that, nuget will also explore that directory when looking for packages.
```

Comment:
typo: downlaod

I was actually thinking of adding this info elsewhere:
E.g. python has details on how to install "daily build" packages directly here: https://packages.grpc.io/archive/2018/08/708288ebc7f8a6c040def1958d2cac558925868b-b40db139-8c47-4a16-aaf2-ae986eb1265c/index.xml

Btw, once we release the nugets in gRPC 1.15.0, the section how to install from local nuget source will get obsolete.

Reply:
since is temporary I would mind to have it there. When the package will be release I can update this.